custom/icds_reports/tasks.py

```diff
@@ -641,6 +641,15 @@ def prepare_excel_reports(config, aggregation_level, include_test, beta, locatio
             loc_level=aggregation_level,
             show_test=include_test
         ).get_excel_data(location)
+        # as DatabaseColumn from corehq.apps.reports.sqlreport doesn't format None
+        if aggregation_level > 4 and NUM_LAUNCHED_AWCS in excel_data[0][1][0] and beta:
+            num_launched_awcs_column = excel_data[0][1][0].index(NUM_LAUNCHED_AWCS)
+            for record in [record for n, record in enumerate(excel_data[0][1]) if n > 0]:
```

Comment:
This can be rewritten much clearer as 
`for record in excel_data[0][1][1:]:`

Reply:
Thanks, great catch.